force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -29,6 +29,12 @@ public with sharing class FlowControlLoopsCollections {
      * @return "Hello World!" if x greater than y, otherwise return null.
      */
     public static String helloWorld(Integer x, Integer y) {
+        String hw = 'Hello World!';
+
+        if (x > y) {
+            return hw;
+        }
+
```

Comment:
Good work here My  suggestion is to skip creating the variable hw. If all you are going to do is return it it just take space in your code and in memory. 

 if (x > y) {
            return 'Hello World';
        }